generators/app/templates/client/public/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title><%= _.kebabCase(appName) %></title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script src="/bundle.js"></script>
```

Comment:
Yes, I've forgotten to change that. There are some problems with the WEBPACK_DEV_SERVER_URL and the publicPath related to it.